index.html

```diff
@@ -160,7 +160,47 @@ <h3><dfn>CropTarget</dfn> Definition</h3>
             <a href="https://github.com/w3c/mediacapture-region/issues/18">issue #18</a>.
           </p>
         </div>
-        <dl data-link-for="CropTarget" data-dfn-for="CropTarget"></dl>
+        <dl data-link-for="CropTarget" data-dfn-for="CropTarget">
+          <dt>
+            <dfn>fromElement()</dfn>
+          </dt>
+          <dd>
+            <p>
+              Calling {{CropTarget/fromElement}} with an {{Element}} of a supported type associates
+              that {{Element}} with a {{CropTarget}}. This {{CropTarget}} may be used as input to
+              {{BrowserCaptureMediaStreamTrack/cropTo}}. We define a
+              <dfn>valid CropTarget</dfn> as one returned by a previous call to
+              {{CropTarget.fromElement()}} in the current [=top-level browsing context=] or any of
+              its
+              <a data-cite="HTML#list-of-the-descendant-browsing-contexts"
+                >descendant browsing contexts</a
+              >.
+            </p>
+            <p>
+              When {{CropTarget/fromElement}} is called with a given <var>element</var>, the user
+              agent [=create a CropTarget|creates a CropTarget=] with <var>element</var> as input.
+              The user agent MUST return a {{Promise}} <var>p</var>. The user agent MUST resolve
+              <var>p</var> only after it has finished all the necessary internal propagation of
+              state associated with the new {{CropTarget}}, at which point the user agent MUST be
+              ready to receive the new {{CropTarget}} as a valid parameter to
+              {{BrowserCaptureMediaStreamTrack/cropTo}}.
+            </p>
+            <p>
+              When cloning an {{Element}} on which {{CropTarget/fromElement}} was previously called,
+              the clone is not associated with any {{CropTarget}}. If {{CropTarget/fromElement}} is
+              later called on the clone, a new {{CropTarget}} will be assigned to it.
+            </p>
+            <div class="note">
+              <p>
+                There is no consensus yet on the following issue: Whether producing a {{CropTarget}}
```

Comment:
Might it be simpler to say
```suggestion
                There is no consensus yet on whether producing a {{CropTarget}}
```
?